docset/winserver2022-ps/grouppolicy/Backup-GPO.md

```diff
@@ -152,14 +174,15 @@ Accept wildcard characters: False
 ```
 
 ### -Guid
+
 Specifies the GPO to backup by its globally unique identifier (GUID).
-The GUID uniquely identifies the GPO.
+The `GUID` uniquely identifies the GPO.
```

Comment:
Please reflow short lines and long lines.

Reply:
I'll look at this and update the PR